src/fs_event_wrap.cc

```diff
@@ -111,7 +111,10 @@ void FSEventWrap::Start(const FunctionCallbackInfo<Value>& args) {
 
   FSEventWrap* wrap;
   ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
-  CHECK_EQ(wrap->initialized_, false);
+  if (wrap->initialized_) {
+    args.GetReturnValue().Set(0);
+    return;
```

Comment:
For consistency with the surrounding code, maybe write this as `return args.GetReturnValue().Set(0);` without braces.